tests/test_workspace.py

```diff
@@ -77,6 +77,21 @@ def test_get_measurement_nonexist(workspace_factory):
     assert 'nonexistent_measurement' in str(excinfo.value)
 
 
+def test_get_measurement_index_outofbounds(workspace_factory):
+    w = workspace_factory()
+    with pytest.raises(pyhf.exceptions.InvalidMeasurement) as excinfo:
+        w.get_measurement(measurement_index=9999)
+    assert 'out of bounds' in str(excinfo.value)
+
+
+def test_get_measurement_no_measurements_defined(workspace_factory):
+    w = workspace_factory()
+    w.measurement_names = []
+    with pytest.raises(pyhf.exceptions.InvalidMeasurement) as excinfo:
+        w.get_measurement()
```

Comment:
```suggestion
    ws = workspace_factory()
    with pytest.raises(pyhf.exceptions.InvalidMeasurement) as excinfo:
        ws.get_measurement(measurement_index=9999)
    assert 'out of bounds' in str(excinfo.value)


def test_get_measurement_no_measurements_defined(workspace_factory):
    ws = workspace_factory()
    ws.measurement_names = []
    with pytest.raises(pyhf.exceptions.InvalidMeasurement) as excinfo:
        ws.get_measurement()
```

nitpicky annoying request for no single letter variables.

Though as I realize there are other instances where this occurs in this test file and other instances that use `ws` so if you'd prefer I just make a harmonizing PR I can do that too.